flink-yarn/src/test/java/org/apache/flink/yarn/RegisterApplicationMasterResponseReflectorTest.java

```diff
@@ -125,24 +119,22 @@ public void testDoesntCallGetSchedulerResourceTypesMethodIfAbsent() {
                 registerApplicationMasterResponseReflector.getSchedulerResourceTypeNamesUnsafe(
                         new Object());
 
-        assertFalse(schedulerResourceTypeNames.isPresent());
+        assertThat(schedulerResourceTypeNames).isNotPresent();
     }
 
     @Test
-    public void testGetSchedulerResourceTypesMethodReflectiveHadoop26() {
+    void testGetSchedulerResourceTypesMethodReflectiveHadoop26() {
         assumeTrue(
+                isHadoopVersionGreaterThanOrEquals(2, 6),
```

Comment:
could be removed since we use a newer version nowadays

Reply:
Done.